EpiAware/docs/src/showcase/replications/chatzilena-2019/index.jl

```diff
@@ -0,0 +1,622 @@
+
+We downloaded the data of this outbreak using the R package `outbreaks` which is maintained as part of the [R Epidemics Consortium(RECON)](http://www. repidemicsconsortium.org).
+
+"
+
+# ╔═╡ 7c9cbbc1-71ef-4d81-b93a-c2b3a8683d53
+data = "https://raw.githubusercontent.com/CDCgov/Rt-without-renewal/refs/heads/446-add-chatzilena-et-al-as-a-replication-example/EpiAware/docs/src/showcase/replications/chatzilena-2019/influenza_england_1978_school.csv2" |>
+       url -> CSV.read(download(url), DataFrame) |>
+              df -> @transform(df,
+    :ts=(:date .- minimum(:date)) .|> d -> d.value + 1.0,)
+
+# ╔═╡ aba3f1db-c290-409c-9b9e-6065935ede54
+N = 763;
+
+# ╔═╡ bb07a580-6d86-48b3-a79f-d2ed9306e87c
+sir_prob = ODEProblem(
+    sir!,
+    N .* [0.99, 0.01, 0.0],
+    (0.0, (Date(1978, 2, 4) - Date(1978, 1, 22)).value + 1)
+)
+
+# ╔═╡ 3f54bb44-76c4-4744-885a-46dedfaffeca
+md"
+## Inference for the deterministic SIR model
+
+The boarding school data gives the number of children \"in bed\" and \"convalescent\" on each of 14 days from 22nd Jan to 4th Feb 1978. We follow _Chatzilena et al_ and treat the number \"in bed\" as a proxy for the number of children in the infectious (I) compartment in the ODE model.
+
+The full observation model is:
+
+```math
+\begin{aligned}
+Y_t &\sim \text{Poisson}(\lambda_t)\\
+\lambda_t &= I(t)\\
+\beta &\sim \text{LogNormal}(\text{logmean}=0,\text{logstd}=1) \\
+\gamma & \sim \text{Gamma}(\text{shape} = 0.004, \text{scale} = 50)\\
+S(0) /N &\sim \text{Beta}(0.5, 0.5).
+\end{aligned}
+```
+
+**NB: Chatzilena et al give $\lambda_t = \int_0^t  \beta \frac{I(s)}{N} S(s) - \gamma I(s)ds = I(t) - I(0).$ However, this doesn't match their underlying stan code.**
+"
+
+# ╔═╡ ea1be94b-d722-47ee-8465-982c83dc6838
+md"
+From `EpiAware`, we have the `PoissonError` struct which defines the probabilistic structure of this observation error model.
+"
+
+# ╔═╡ 87509792-e28d-4618-9bf5-e06b2e5dbe8b
+obs = PoissonError()
+
+# ╔═╡ 81501c84-5e1f-4829-a26d-52fe00503958
+md"
+Now we can write the observation model using the `Turing` PPL.
+"
+
+# ╔═╡ 1d287c8e-7000-4b23-ae7e-f7008c3e53bd
+@model function deterministic_ode_mdl(Yt, ts, obs, prob, N;
+        solver = AutoTsit5(Rosenbrock23()),
+        upjitter = 1e-3
+)
+    ##Priors##
+    β ~ LogNormal(0.0, 1.0)
+    γ ~ Gamma(0.004, 1 / 0.002)
+    S₀ ~ Beta(0.5, 0.5)
+
+    ##remake ODE model##
+    _prob = remake(prob;
+        u0 = [S₀, 1 - S₀, 0.0],
+        p = [β, γ]
+    )
+
+    ##Solve remade ODE model##
+
+    sol = solve(_prob, solver;
+        saveat = ts,
+        verbose = false)
+
+    ##log-like accumulation using obs##
+    λt = N * sol[2, :] .+ upjitter #expected It
```

Comment:
is the jitter needed here I thought we had handled that with the obserror types?

Reply:
No not the negative safety (unless I missed something v possbile)